src/GraphQl/Serializer/SerializerContextBuilder.php

```diff
@@ -78,6 +79,11 @@ private function fieldsToAttributes(?string $resourceClass, ?ResourceMetadata $r
             $fields = $info->getFieldSelection(PHP_INT_MAX);
         }
 
+        // On lowest symfony dependencies there's a bug in the name converter not accepting a string in the group ssee https://github.com/symfony/symfony/pull/34490
```

Comment:
```suggestion
        // On lowest symfony dependencies there's a bug in the name converter not accepting a string in the group see https://github.com/symfony/symfony/pull/34490
```

I suggest to require `symfony/serializer` >=4.3.9 instead.